packages.md

```diff
@@ -143,6 +143,16 @@ provide nice nicknames.
 
 Many implementations (SBCL, CCL, ECL, Clasp, ABCL, ACL, LispWorks >= 7.2…) support Package Local Nicknames (PLN).
 
+To use a `PLN` you can simply do the following, for example, if you'd like to try out a `PLN` in an ad-hoc fashion:
+
+~~~lisp
+(add-package-local-nickname :a :alexandria)
```

Comment:
it's in uiop

Reply:
I'll add it today and tag you when I push. Thanks!